flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/docs/CrdReferenceDoclet.java

```diff
@@ -210,7 +233,7 @@ public Void scan(Element e, Integer depth) {
                 case ENUM_CONSTANT:
                     out.println(
                             "| "
-                                    + e
+                                    + getJsonPropValueOfEnum(e)
```

Comment:
I think this logic should be applied to all fields, not only enums. Currently only enums are affected but we could define @JsonProperty annotations on other fields in the future